docs/source/getting-started/installing.adoc

```diff
@@ -128,4 +128,4 @@ $ rm -rf ~/.kube
 
 .  With your hypervisor management tool, confirm that there are no
 remaining artifacts related to the Minishift VM. For example, if you use
-KVM, you need to run the `virsh` command.
+KVM, you need to run the `virsh` command to avoid the issue link:../using/troubleshooting{outfilesuffix}#domain-minishift-already-exists[Error starting the VM: ...operation failed: domain 'minishift' already exists...].
```

Comment:
@Preeticp I do not think this is required. 914172
Users who want to know uninstall  steps for Minishift should not get redirected to  troubleshooting issue `Error starting the VM: ...operation failed: domain 'minishift' already exists`.